File: 2023/domaci/45-cteni-souboru/volfmatous/main.js
const readline = require("readline");
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
    terminal: false,
});
const fs = require("fs");

let linesRead = 0;
const lines = [];

async function readLine() {
    if (lines.length < 1) {
        for await (const line of rl) {
            lines.push(line);
        }
    }

    linesRead++;
    return lines[linesRead - 1];
}

main().then();

async function main() {
    const filePath = await readLine();

    if (!fs.existsSync(filePath)) {
        console.log("not found");
        return;
    }

    const content = fs.readFileSync(filePath, "utf8");
    console.log(content);
}

File: 2023/domaci/45-cteni-souboru/volfmatous/main.cs
using System;
using System.IO;

internal class Program
{
    static void Main(string[] args)
    {
        string filePath = Console.ReadLine();

        if (!File.Exists(filePath))
        {
            Console.WriteLine("not found");
            return;
        }

        string content = File.ReadAllText(filePath);
        Console.WriteLine(content);
    }
}
